### PR TITLE
OTJ: Tomb Trawler, Trash the Town, Treasure Dredger

### PR DESCRIPTION
**Pull request by dracontes**
Tested card scripts for:
- [Tomb Trawler](https://scryfall.com/card/otj/250/tomb-trawler)
- [Trash the Town](https://scryfall.com/card/otj/186/trash-the-town)
- [Treasure Dredger](https://scryfall.com/card/otj/110/treasure-dredger)

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/trash_the_town.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Trash the Town
+ManaCost:G
+Types:Instant
+K:Spree
+A:SP$ Charm | Choices$ DBCounters,DBTrample,DBDrawTwo | MinCharmNum$ 1 | CharmNum$ 3 | Spree$ True
+SVar:DBCounters:DB$ PutCounter | SpreeCost$ 2 | ValidTgts$ Creature | TgtPrompt$ Select target creature | CounterType$ P1P1 | CounterNum$ 2 | SpellDescription$ Put two +1/+1 counters on target creature.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/trash_the_town.txt:6]: these kind of simple TgtPrompts shouldn't be needed

File: forge-gui/res/cardsfolder/upcoming/trash_the_town.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+SVar:DBTrample:DB$ Pump | SpreeCost$ 1 | ValidTgts$ Creature | TgtPrompt$ Select target creature | KW$ Trample | SpellDescription$ Target creature gains trample until end of turn.
+SVar:DBDrawTwo:DB$ Pump | SpreeCost$ 1 | ValidTgts$ Creature | TgtPrompt$ Select target creature | SubAbility$ DBAnimate | SpellDescription$ Until end of turn, target creature gains "Whenever this creature deals combat damage to a player, draw two cards."
+SVar:DBAnimate:DB$ Animate | Triggers$ TrigCDPlayer | Defined$ ParentTarget | StackDescription$ None
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/trash_the_town.txt:9]: why not target directly with Animate?

File: forge-gui/res/cardsfolder/upcoming/trash_the_town.txt (hunk 1 above)
Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/trash_the_town.txt]: Sure enough they aren't. Will keep that in mind going forward.

File: forge-gui/res/cardsfolder/upcoming/trash_the_town.txt (hunk 2 above)
Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/trash_the_town.txt]: That does work 👍 
Here I referenced Feign Death, mostly to make sure I wasn't skipping steps. I take it then Feign Death's script and similar ones could use a cleanup?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/trash_the_town.txt]: sure

**Review by tool4ever — APPROVED** [review]

**Pull request merged**